File: dbpro/src/main/java/feat/employeetask/EmployeeTaskDAO.java
package feat.employeetask;

import dao.admin.DAO;
import oracle.connect.JDBCManager;

public class EmployeeTaskDAO implements DAO {

	private EmployeeTaskDAO() {
	}

	private static EmployeeTaskDAO instance;

	public static EmployeeTaskDAO getInstance() {
		if (instance == null) {
			instance = new EmployeeTaskDAO();
		}
		return instance;
	}

	@Override
	public void selectEmployees() {
		final String query = "SELECT " +
			"	EMPLOYEE_ROLE, " +
			"	EMPLOYEE_TASK_CON, " +
			"	EMPLOYEE_TASK_SAL " +
			"FROM " +
			"	EMPLOYEE_TASK";

		JDBCManager
			.getInstance()
			.queryForList(query, EmployeeTaskDTO.class)
			.forEach(employeeTaskDTO -> System.out.println(employeeTaskDTO.toString()));
	}

}

File: dbpro/src/main/java/kr/ac/sejong/feat/employeetask/EmployeeTaskDAO.kt
package kr.ac.sejong.feat.employeetask

import kr.ac.sejong.config.JDBCManager

class EmployeeTaskDAO private constructor() {
    companion object {
        @JvmStatic
        var instance: EmployeeTaskDAO? = null
            get() {
                if (field == null) {
                    field = EmployeeTaskDAO()
                }
                return field
            }
            private set
    }

    fun selectEmployeeTasks(): List<EmployeeTaskDTO> {
        val query = """
            SELECT
                EMPLOYEE_ROLE,
                EMPLOYEE_TASK_CON,
                EMPLOYEE_TASK_SAL
            FROM
                EMPLOYEE_TASK
        """

        return JDBCManager
            .getInstance()
            .queryForList(query, EmployeeTaskDTO::class.java)
    }

}